lib/puppet/graph/simple_graph.rb

```diff
@@ -228,23 +228,19 @@ def report_cycles_in_graph
     number_of_cycles = cycles.length
     return if number_of_cycles == 0
 
-    cycle_path_list = ''
+    message = n_("Found %{num} dependency cycle:\n", "Found %{num} dependency cycles:\n", number_of_cycles) % { num: number_of_cycles }
```

Comment:
Is that an errant `n` before the `_(`?

Reply:
No, that's how you do proper pluralization. Notice there are two versions of the message, one singular, one plural, and it picks one based on the number passed in.

Reply:
oh cool! I was wondering about that. Thank you for the clarification!